numpy/ma/core.py

```diff
@@ -25,6 +25,7 @@
 import sys
 import operator
 import warnings
+import textwrap
```

Comment:
This is already imported elsewhere in numpy based on looking at `sys.modules`, so shouldn't impact startup time